### DISCUSSION TITLE
Cloud: Handling Dev/Prod/Multi-Tenant?

### DISCUSSION THREAD

**Discussion by MentalGear** [directus/directus#16533] [Archive]
I was wondering how I can keep Dev and Production Directus "Projects" in sync  in directus cloud?

Example:
On a dev project with dev test data: I add a new field and  a new automation.
Now I want the new "structure" synced with my production project that has different data. How can I transfer the structure to the production project?

PS: I guess in a broader sense this asks about transferring/applying all of "directus" from a "master" - without the data - to another directus project (which could also be used to have multi-tendency across various isolated project instances).

**Comment by br41nslug** [comment]
You can use snapshots to migrate the schema without data to a new environment :) https://docs.directus.io/self-hosted/cli.html#snapshot-the-data-model

**Reply by MentalGear** [reply]
Yeah I noticed, but does this work on directus cloud as well?

**Reply by br41nslug** [reply]
Ahh sorry i had absolutely read over the cloud part 🤦 No this is currently not available on the cloud

**Reply by MentalGear** [reply]
Any wide estimates when this (or raw db access in general) might come to cloud?
Would really like to take advantage of your amazing offering, but it's too limiting right now when you need a multi-tenant db structure, like "Organisation > Teams > Projects". I'd hoped by having a "cloud project" for each organisation and doing migrations from a "master structure directus instance" I could realize this multi-tenancy structure.

**Reply by wfriedman89** [reply]
@MentalGear Directus Enterprise Cloud allows for Direct DB access as well as use of the schema migrate tool that @br41nslug mentioned above.  We have many customers who migrate data from one instance to other(s).  If you are using Standard Cloud, you can upgrade to Enterprise CloudIf you are interested in learning more about Directus Enterprise Cloud, please reach out to us here:  https://directus.io/contact/
